main.js

```diff
@@ -0,0 +1,77 @@
+import $ from 'jquery'
+import './style/style.sass'
+import './style/reset.css'
+import { Todo } from './Todo.js'
+
+const inputText = $('.input-text')
+const buttonAddTodo = $('.add-item')
+const todoList = $('.todo__list')
+const filtrAll = $('.filtr-all')
+const filtrTodo = $('.filtr-todo')
+const filtrDone = $('.filtr-done')
+
+buttonAddTodo.click(() => {
+  if (inputText.val() === '') {
+    alert('Please write any text')
+
+    return
+  }
+
+  const todo = new Todo(inputText.val())
+  todo.createNewTodo().appendTo(todoList)
+
+  const acceptedButton = todo.getAcceptButton()
+  const deleteButton =  todo.getDeleteButton()
+
+  acceptedTodo(acceptedButton, todo)
+  deleteTodo(deleteButton)
+
+  inputText.val(null)
+})
+
+const acceptedTodo = (acceptedButton, todo) => {
+  acceptedButton.click(event => {
+    if ($(event.target).closest('li').hasClass('checked')) {
```

Comment:
`if (todo.isDone)`, you have eveyrthing in the data model, so why don't you use it